Docs(qBitrr): update installation-notes.md

Tanguille opened the pull request:
**Description**

Added some explanation since I failed to configure qBitrr properly following this installation-notes. I assume something changed upstream which added a `.config` directory inside the `config` directory which now both have `config.toml` but only the one in `.config` actually applying. However if @kqmaverick installs the chart he doesn't seem to get the `.config` directory while I did on 2 instances I tested. I guess some more testing is required on why this is happening.

⚒️ Fixes outdated installation-notes

**⚙️ Type of change**

- [ ] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**
<!--
I double checked with 2 instances and asked community feedback on their experience.
-->

**📃 Notes:**
<!-- Please enter any other relevant information here -->

**✔️ Checklist:**

- [X] ⚖️ My code follows the style guidelines of this project
- [X] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [X] 📄 I have made corresponding changes to the documentation
- [ ] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [ ] ⬆️ I increased versions for any altered app according to semantic versioning

**➕ App addition**

If this PR is an app addition please make sure you have done the following.

- [ ] 🪞 I have opened a PR on [truecharts/containers](https://github.com/truecharts/containers) adding the container to TrueCharts mirror repo.
- [ ] 🖼️ I have added an icon in the Chart's root directory called `icon.png`

---

Tanguille committed:
Update setup-guide.md wg-easy

Added Caution that the domain used  may not be a behind proxy because I had that exact issue after setting up wg-easy based on this guide.

Signed-off-by: Tanguille <91473554+Tanguille@users.noreply.github.com>

Tanguille committed:
Merge branch 'truecharts:master' into master

StevenMcElligott committed:
Update charts/stable/wg-easy/docs/setup-guide.md

Signed-off-by: StevenMcElligott <89483932+StevenMcElligott@users.noreply.github.com>

Tanguille committed:
Merge branch 'truecharts:master' into master

Tanguille committed:
docs: update installation-notes.md

Added some explanation since I failed to configure qBitrr properly following this installation-notes.

Signed-off-by: Tanguille <91473554+Tanguille@users.noreply.github.com>

kqmaverick commented:
The config.toml location changes based on a bluefin or cobia install. Any possible insights @stavros-k ?

Bluefin = /config/config.toml
Cobia = /config/.config/config.toml

Tanguille commented:
Do I adjust this PR to match #14103?

kqmaverick commented:
Yes you can remove all the /.config stuff

Tanguille committed:
docs(qBitrr): adjust path according to #14103

The .config directory will no longer exists once #14103 is merged hence I adjusted the installation notes according to this.

Signed-off-by: Tanguille <91473554+Tanguille@users.noreply.github.com>

stavros-k commented on charts/stable/qbitrr/docs/installation-notes.md:
```diff
@@ -32,11 +32,11 @@ If `CompletedDownloadFolder` option is not set to `"/completed_downloads"` qBitr
 ### qBit
 
-This section has qBittorrent specific settings. Ensure that `Host`, `Port`, `UserName`, and `Password` all match your qBittorrent installation. The host and port can be located using the Heavyscript `heavyscript dns -a` command.
+This section has qBittorrent specific settings. Ensure that `Host`, `Port`, `UserName`, and `Password` all match your qBittorrent installation. The host and port can be located using the Heavyscript `heavyscript dns -a` command. It is not recommended to use internal links for this part.
```
Why it is not recommended?

kqmaverick replied:
qBittorrent internal linking works fine.

Tanguille replied:
Good point, reading the installation notes I got that impression since I didn't understand how internal linking worked. My bad, I'll fix it.

Tanguille committed:
docs(qbitrr): fix misunderstanding internal linking

Signed-off-by: Tanguille <91473554+Tanguille@users.noreply.github.com>

Tanguille committed:
docs(qbitrr): fix mistake duplicate

Signed-off-by: Tanguille <91473554+Tanguille@users.noreply.github.com>

Tanguille committed:
docs(qbitrr): fix typo

Signed-off-by: Tanguille <91473554+Tanguille@users.noreply.github.com>

stavros-k commented:
@all-contributors please add @Tanguille  for docs

truecharts-admin commented:
This PR is locked to prevent necro-posting on closed PRs. Please create a issue or contact staff on discord if you want to further discuss this
